Apps/System/music.c

```diff
@@ -0,0 +1,461 @@
+/* music.c
+ *
```

Comment:
I think local style is:

```
/* music.c
 * Music player
 * RebbleOS
 *
 * Author: ...
```

Reply:
Also, having a longer-form comment here describing what's going on and how this works would be good.  (A few sentences summarizing the general design and architecture would help the reader quite a lot.)